docs/08-hypertable/overview.md

```
@@ -0,0 +1,153 @@
+    where=[("_status", "eq", "error")],
+    include_status=True,
+)
+# [{'page_id': 'p3', '_status': 'error', '_error': 'RateLimitError: ...'}]
+
+# Retry — only the errored page re-runs, successful pages are skipped
+docs.insert(doc_id="d1", text="...")
+```
+
+## How It Fits Together
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)